.cirrus.yml

```diff
@@ -67,6 +67,29 @@ build_task:
   cleanup_before_cache_script:
     - cleanup_maven_repository
 
+mend_scan_task:
+  depends_on:
+    - build
+  eks_container:
+    <<: *EKS_CONTAINER
+    cpu: 2
+    memory: 2G
+  # run only on master and long-term branches
+  only_if: $CIRRUS_USER_COLLABORATOR == 'true' && ($CIRRUS_BRANCH == "master" || $CIRRUS_BRANCH =~ "branch-.*")
+  env:
+    WS_APIKEY: VAULT[development/kv/data/mend data.apikey]
+  maven_cache:
+    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
+  whitesource_script:
+    - source cirrus-env QA
+    - source set_maven_build_version $BUILD_NUMBER
+    - mvn clean install -DskipTests --batch-mode
+    - source ws_scan.sh
+  allow_failures: "true"
```

Comment:
[Our documentation](https://xtranet-sonarsource.atlassian.net/wiki/spaces/DEV/pages/2611314731/Configuring+Mend+Scan) mentions `cleanup_before_cache_script: cleanup_maven_repository` (Script [here](https://github.com/SonarSource/re-ci-images/blob/master/docker/bin/src/cleanup_maven_repository)) after the whitesource_script. 
Is it needed here?

Reply:
No idea, but I have added it :)